kratos/processes/skin_detection_process.cpp

```diff
@@ -304,19 +304,20 @@ void SkinDetectionProcess<TDim>::SetUpAdditionalSubModelParts(const ModelPart& r
                 std::vector<IndexType> conditions_ids_buffer;
 
                 // We iterate over the nodes of this model part
-                auto& sub_nodes_array = sub_model_part.Nodes();
+                auto& r_sub_nodes_array = r_sub_model_part.Nodes();
+                const auto it_node_begin = r_sub_nodes_array.begin();
                 #pragma omp for
```

Comment:
maybe an opportunity to move this to  parallel utilities?

Reply:
I know, but I am afraid to break it :P (could be in a following PR?)

Reply:
ok